src/rev_ai/apiclient.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+"""Speech recognition tools for using rev.ai
+"""
+try:
+    from urllib.parse import urljoin
+except ImportError:
+    from urlparse import urljoin
+import os
+import requests
+import logging
+import json
+from models import (
```

Comment:
> I was hoping to find a better way to import the classes I wanted using the init.py in the models folder. Is there none?

https://www.python.org/dev/peps/pep-0020/ says:
Explicit is better than implicit.